src/arfsdao.ts

```diff
@@ -209,12 +217,13 @@ export class ArFSDAO extends ArFSDAOAnonymous {
 		super(arweave, appName, appVersion);
 	}
 
-	async createPublicFolder(
-		folderName: string,
-		driveId: DriveID,
-		parentFolderId?: FolderID,
+	async createPublicFolder({
+		folderData,
```

Comment:
Interesting that only create folder re-uses the data object created for estimation.